stock_barcodes_picking_batch/views/stock_picking_batch_views.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="stock_batch_picking_form" model="ir.ui.view">
+        <field name="model">stock.picking.batch</field>
+        <field name="inherit_id" ref="stock_picking_batch.stock_picking_batch_form" />
+        <field name="arch" type="xml">
+            <div class="oe_title" position="before">
+                <div class="oe_button_box" name="button_box">
+                    <button
+                        name="action_barcode_scan"
+                        class="oe_stat_button"
+                        icon="fa-barcode"
+                        type="object"
+                        help="Start barcode interface"
+                        states="in_progress"
+                    >
+                        <div class="o_form_field o_stat_info">
+                            <span class="o_stat_text">Scan barcodes</span>
+                        </div>
+                    </button>
+                </div>
```

Comment:
**issue**: adding the button_box here could clash with other modules that do the same, e.g.
https://github.com/OCA/l10n-italy/blob/14.0/l10n_it_delivery_note_batch/views/stock_picking_batch_views.xml#L28-L43

I'm not sure how it's rendered in the end.

An alternative could be to just have a regular button in the header, instead of using the button box

Reply:
Love the idea, I moved the button to the header